### PR TITLE
Adds a Follow/Jump to link to the VV dropdown

### PR DESCRIPTION
**Pull request by tkdrg**
Some good QoL for debugging stuff.

### CODE REVIEW

**Comment by duncathan** [comment]
Can you add a way to orbit the object, too? For things that are moving

**Comment by MrStonedOne** [comment]
follow is orbit

**Comment by duncathan** [comment]
Yeah, I'm asking if he can add a way to orbit objects, not just mobs.

**Comment by tkdrg** [comment]
It is done

**Comment by duncathan** [comment]
Sweet thanks

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -243,6 +243,11 @@ datum/proc/on_varedit(modified_var) //called whenever a var is edited
 	body += "<option value='?_src_=vars;proc_call=\ref[D]'>Call Proc</option>"
 	if(ismob(D))
 		body += "<option value='?_src_=vars;mob_player_panel=\ref[D]'>Show player panel</option>"
+	if(ismob(D) || isobj(D))
```
Comment (MrStonedOne) [inline code/datums/datumvars.dm]: This should be a check for atom/movable, not mob or obj, same thing, but makes it only 1 check.

Comment (tkdrg) [inline code/datums/datumvars.dm]: is there a built-in isam() proc or something like that

**Comment by tkdrg** [comment]
okay done

**Pull request merged**